R/fimsframe.R

```diff
@@ -61,9 +64,18 @@ setMethod("ages", "FIMSFrame", function(x) x@ages)
 setGeneric("n_ages", function(x) standardGeneric("n_ages"))
 setMethod("n_ages", "FIMSFrame", function(x) x@n_ages)
 
+setGeneric("comp_lengths", function(x) standardGeneric("comp_lengths"))
+setMethod("comp_lengths", "FIMSFrame", function(x) x@comp_lengths)
```

Comment:
Again, why "comp_lengths" instead of just "lengths"?

Reply:
I think some of the issue here is that we are using ages as a population dimension as well as the ages for composition data. I think eventually it would be better to have population_ages and comp_ages as separate inputs, then comp_ages can be fleet specific.

Reply:
everything is now comp_lengths and comp_ages so they are consistent.